en/editor/publish/publish-web.md

```diff
@@ -55,12 +56,22 @@ Next, click the **Run** button to open the built game version in the browser for
 
 The picture above is the preview of the Web Mobile platform. Notice that the game view occupies the entire browser window, while the game view of Web Desktop has a fixed resolution and will not fill the screen.
 
+### With WebGPU
+
+Cocos Creator support WebGPU as rendering backend for building Web-Desktop since 3.6.2, just enable `WebGPU` when building Web-Desktop.
```

Comment:
```suggestion
[WebGPU](https://www.w3.org/TR/webgpu/) is the next generation GPU API for the web. Cocos Creator supports WebGPU as rendering backend for building Web-Desktop since 3.6.2, just enable `WebGPU` option in the build panel when building Web-Desktop.
```